tests/end-to-end/tests/components/menu.cy.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2022 Dynamic Solutions
```

Comment:
Isn't it 2023 ?:)

Reply:
Yep :). Opened https://github.com/websight-io/bulma/pull/156 to update it. Thanks!